Makefile.am

```diff
@@ -158,14 +158,66 @@ base_files = \
 bin_PROGRAMS += csd
 
 csd_SOURCES = \
-	${base_files} \
+	${BASE_FILES} \
```

Comment:
checking now... you actually don't have a different .o per target. Kind of weird because if you have a different flag it won't apply, but I think it's ok for now.